{"timestamp":"2025-09-26T14:19:25.549297","level":"warning","event":"Could not import pandas. Holidays will not be considered.","logger":"airflow.example_dags.plugins.workday"}
{"timestamp":"2025-09-26T14:19:25.688513","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-26T14:19:25.689957","level":"info","event":"Filling up the DagBag from dags/update_db.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-26T14:19:31.140393Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:31.141697Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:31.142094Z","level":"info","event":"Current task name:load_core","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:31.142286Z","level":"info","event":"Dag name:update_db","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:34.636491","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AttributeError","exc_value":"'Connection' object has no attribute 'commit'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/.venv/lib/python3.11/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/Users/meissagningue/Projects/dev_ia/pipeline_etl_youtube/dags/update_db.py","lineno":140,"name":"load_core"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-26T14:19:35.923023Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:35.926058Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:35.926532Z","level":"info","event":"Task:<Task(PythonOperator): load_core>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-26T14:19:35.927068Z","level":"info","event":"Failure caused by 'Connection' object has no attribute 'commit'","chan":"stdout","logger":"task"}
